### PR TITLE
Resend verification email

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/components/cards/SightingsCard.jsx [hunk 1]
```diff
@@ -55,36 +55,36 @@ export default function SightingsCard({
...
+      labelId: 'DATE_OF_SIGHTING',
       options: {
         cellRenderer: cellRendererTypes.specifiedTime,
       },
     },
     {
       reference: 'created',
       name: 'created',
-      label: 'Date created',
+      labelId: 'DATE_REPORTED',
```
Comment (Atticus29) [inline src/components/cards/SightingsCard.jsx:66]: This and DATE_OF_SIGHTING look like their order was swapped. Just confiming that this was intentional.

[the remark this answers is not in the document]
Comment (Atticus29) [reply src/components/cards/SightingsCard.jsx:66]: Incidentally, I'm curious why we are opting not to display this?
columns = ['date', 'location', 'actions']

**Review by Atticus29 — APPROVED** [review]
Approved with minor question/comments.

Also, going forward, if a branch is linked to a JIRA ticket, could that information be provided? I hadn't realized that this or its predecessor were linked to DEX-830 until Emily mentioned that it was missing a few things (which raises the question of how she knew they were connected).

File: src/components/cards/SightingsCard.jsx [hunk 2]
```diff
@@ -55,36 +55,36 @@ export default function SightingsCard({
...
+      labelId: 'LOCATION',
       options: {
         cellRenderer: cellRendererTypes.location,
       },
     },
     {
       reference: 'locationIdValue',
       name: 'locationId_value',
-      label: 'Location',
+      labelId: 'LOCATION',
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/components/cards/SightingsCard.jsx:82]: ah this is for real! sometimes the backend gives us locationIds by guid, other times they give us the label directly and we dont have to parse.

File: src/components/cards/SightingsCard.jsx (hunk 1 above)
Comment (brmscheiner) [reply src/components/cards/SightingsCard.jsx:66]: i suppose we could! worth asking if it would be nice to include

**Comment by brmscheiner** [comment]
@Atticus29 yes, i will try to remember to include the jira tickets in the description

**Pull request merged**